S1_NRB/metadata/extract.py

```diff
@@ -14,13 +14,13 @@
 from spatialist.raster import rasterize
 from osgeo import gdal
 import S1_NRB
-from S1_NRB.metadata.mapping import NRB_PATTERN, LERC_ERR_THRES, RES_MAP, ORB_MAP, DEM_MAP, SLC_ACC_MAP
+from S1_NRB.metadata.mapping import ARD_PATTERN, LERC_ERR_THRES, RES_MAP, OSV_MAP, DEM_MAP, SLC_ACC_MAP
 from S1_NRB import snap
 
 gdal.UseExceptions()
 
 
-def meta_dict(config, target, src_ids, rtc_dir, proc_time, start, stop, compression):
+def meta_dict(config, target, src_ids, sar_dir, proc_time, start, stop, compression, orb=False):
```

Comment:
same as in https://github.com/SAR-ARD/S1_NRB/pull/122. Let's make this a generic parameter `product_type`.